ci-operator/step-registry/cucushift/installer/check/vsphere/customized-resource/cucushift-installer-check-vsphere-customized-resource-commands.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+source "${SHARED_DIR}/govc.sh"
+function get_node_names() {
+   local node_type=$1
+   if [ -n "${node_type}" ]; then
+      mapfile -t  node_lists < <(oc get node --no-headers | grep ${node_type} | awk '{print $1}')
+   else
+      mapfile -t  node_lists < <(oc get node --no-headers | awk '{print $1}')
+   fi
+  
+   echo ${node_lists[*]}
+}
+
+function check_master_nodes_number() {
+
+#get node name
```

Comment:
```suggestion
    #get node name
    master_lists=($(get_node_names "master"))
```

Reply:
shell check failed with:

In .//ci-operator/step-registry/cucushift/installer/check/vsphere/customized-resource/cucushift-installer-check-vsphere-customized-resource-commands.sh line 22:
    master_lists=($(get_node_names "master"))
                  ^------------------------^ SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).

Reply:
Then you can add one comment before the line to disable the shell check:
```
# shellcheck disable=SC2207
```